tests/nose2.cfg

```diff
@@ -0,0 +1,27 @@
+[unittest]
+plugins = nose2.plugins.logcapture
+          nose2.plugins.junitxml
+          nose2.plugins.mp
+
+[test-result]
+always-on = True
+descriptions = True
+
+[junit-xml]
+always-on = True
+test_fullname = True
+
+[log-capture]
+always-on = True
+clear-handlers = True
+log-level = DEBUG
+
+[multiprocess]
+always-on = True
+test-run-timeout = 600
+
+[coverage]
+always-on = True
+coverage-report = term \
+                  xml
+cover-package = chsdi
```

Comment:
This is redundant with the .coveragerc file

Reply:
True, it has always been redundant though. In the .coveragerc and also in setup.cfg. When I deleted it from setup.cfg I moved it to nose2.cfg. But I guess once is enough and I'll try if it works when dropping it from nose2.cfg